homeassistant/components/tractive/sensor.py

```diff
@@ -0,0 +1,178 @@
+"""Support for Tractive sensors."""
```

Comment:
You should add yourself as a codeowner :+1:

Reply:
Done